### PR TITLE
Add types declaration file

### LINKED ISSUE #1: TypeScript type declaration file
If you want I can write a TypeScript type declaration file for this package so that TS developers can utilise the type-checking feature of the language as they use this package. Let me know if you are interested.

### PR DESCRIPTION
**Pull request by geopic**
Resolves #1. I typed a number of parameters and return values as `unknown` -- if you want to, you can let me know what they are so I can add them. Otherwise, if you have any other issues, then feel free to say so. Cheers :+1:

### CODE REVIEW

File: omusubi.d.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  }
+  
+  export function defineComponent(tagname: string, options?: ElementDefinitionOptions): (wrapped: typeof Component) => void;
+
+  export function html(strings: string[], values: string[]): new () => RawHTMLTagFuncOutput;
+
+  export class Dispatcher extends Function {
+    constructor(element: unknown, store: unknown);
+    
+    dispatch(action: unknown, ...args: string[]): Promise<void>;
```
Comment (boke0) [inline omusubi.d.ts]: "action" is a **string** which is name of the Dispatchers method.

**Review by boke0 — CHANGES_REQUESTED** [review]
Thank you for making PR!
Most of type defining was correct, but one of the unknown was able to be defined.
Could you please change it?

File: omusubi.d.ts (hunk 1 above)
Comment (geopic) [reply omusubi.d.ts]: Changed :+1:

**Review by boke0 — APPROVED** [review]
LGTM!!!

**Pull request merged**